libraries/core/src/main/kotlin/com/bumble/appyx/core/composable/Children.kt

```diff
@@ -123,6 +125,22 @@ class ChildrenTransitionScope<T : Any, S>(
             transitionDescriptor: TransitionDescriptor<T, S>
         ) -> Unit
     ) {
+        val saveableStateHolder = rememberSaveableStateHolder()
+
+        val removedElementsFlow = remember {
+            navModel.removedElementKeys().map { list ->
+                list
+                    .filter { clazz.isInstance(it.navTarget) }
+            }
+        }
+        LaunchedEffect(this@ChildrenTransitionScope) {
```

Comment:
Why `ChildrenTransitionScope` here and not `navModel`?
Probably we can move flow creating here because it will be invoked almost only once (`navModel` won't change).

Reply:
moved